docs/Example Schedules/CE Schedules/compe-kteal2.md

```diff
@@ -0,0 +1,180 @@
+---
+tags:
+- Robotics
+- Artificial intelligence and machine learning
+---
+# Robotics, Statistics Minor, 8 Semesters
+
+Hi, I'm Keegan, currently a second-semester Junior (CE '25) and I'm pursuing a Statistics minor. I'm focused on software, robotics, autonomy, and controls. Originally, I had planned to use one semester as a co-op but decided to take more technical electives instead. My schedule is definitely on the heavy side but it allows you to spend most of Junior and Senior year taking tech electives that you're interested in, while never having to double up on the big 4 ECE classes.
+
+## Features of My Schedule
+
+- Can be easily modified to 7 semesters (or less)
+- No summer or winter classes, so internship options every summer
+- Covers most of the required classes for MEng in Robotics and Autonomy in undergrad
+- Statistics minor + James Scholar
+
+## Schedule
+
+### AP Credits and Proficiency Tests
+| Course Name | Credit Hours |
+| --- | --- |
+| CHEM 102 | 3 |
+| CHEM 104 | 3 |
```

Comment:
Can you also link all ECE adjacent course reviews?  No need to link geneds.

Reply:
not sure if you want me to add links to pages within the paragraphs after each semesters schedule, but I linked the courses in the schedule itself